specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Create a new spark batch job.",
+        "operationId": "SparkBatch_Create",
+        "x-ms-examples": {
+          "Create a spark batch job": {
+            "$ref": "./examples/HDI_SparkBatch_Create.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/requestedByParameter"
+          },
+          {
+            "name": "sparkBatchJobRequest",
+            "in": "body",
+            "description": "Livy compatible batch job request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJob"
+            }
+          }
+        }
+      }
+    },
+    "/livy/batches/{batchId}": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Gets a single spark batch job.",
+        "operationId": "SparkBatch_Get",
+        "x-ms-examples": {
+          "Gets a single spark batch job": {
+            "$ref": "./examples/HDI_SparkBatch_Get.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "batchId",
+            "in": "path",
+            "description": "Identifier for the batch job.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJob"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Cancels a running spark batch job.",
+        "operationId": "SparkBatch_Delete",
+        "x-ms-examples": {
+          "Cancels a running spark batch job": {
+            "$ref": "./examples/HDI_SparkBatch_Delete.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/requestedByParameter"
+          },
+          {
+            "name": "batchId",
+            "in": "path",
+            "description": "Identifier for the batch job.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Side note (not a Swagger feedback): Usually on a delete with no body the expected response is a 204. This is unusual server behavior here.

Reply:
Fixed it. This request should contain the response message in the body of the response. Added schema for it.